dev-perl/Gearman-XS/Gearman-XS-0.150.0.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+EAPI=5
+MODULE_AUTHOR=KROW
+MODULE_VERSION=0.15
+inherit perl-module
+
+DESCRIPTION='Perl front end for the Gearman C library.'
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+extra_depend() {
+	echo sys-cluster/gearmand
+}
+perl_meta_configure() {
+	# ExtUtils::CBuilder 0.27 ( 0.270.0 )
+	echo \>=virtual/perl-ExtUtils-CBuilder-0.270.0
+	# ExtUtils::MakeMaker 6.59 ( 6.590.0 )
+	echo \>=virtual/perl-ExtUtils-MakeMaker-6.590.0
+	# Missing in meta, added manually
+	echo dev-perl/Module-Install
```

Comment:
Generally speaking, Module::Install dists don't themselves need the Module::Install dependency ( They're bundled in inc/ and its loaded by magic ).

That Gearman does *NOT* do this should be considered a bug upstream.

And indeed, .... https://rt.cpan.org/Ticket/Display.html?id=89690